### PR TITLE
Pkg 4785 2.0.1.2 initial

### PR DESCRIPTION
**Pull request by danpetry**
**Destination channel:** defaults

### Links

- [PKG-4785](https://anaconda.atlassian.net/browse/PKG-4785)

### Explanation of changes:

- Part of CUDA 12.4 build, following [conda-forge's pattern](https://github.com/conda-forge/staged-recipes/issues/21382) which was developed in conjunction with Nvidia. I don't see a need to stage these then release them all at once. They won't cause a problem released one at a time.
- Please check the main branch too, this has just been forked
- Changes made to adjust feedstock for differences between conda-forge and defaults. Please see commit messages for description of changes.

[PKG-4785]: https://anaconda.atlassian.net/browse/PKG-4785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -53,19 +53,21 @@ requirements:
...
     - {{ pin_compatible("cuda-version", max_pin="x") }}
     - cudatoolkit >=11.1,<12  # [(cuda_compiler_version or "").startswith("11")]
     - libcublas               # [(cuda_compiler_version or "").startswith("12")]
   run_constrained:
     # Only GLIBC_2.17 or older symbols present
-    - __glibc >=2.17      # [linux]
+    - __glibc >={{ conda_glibc_ver }}      # [linux]
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
 
 test:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:73]: Should we constrain `git  # [win]` as it's available on our win-64 builders? I'm not sure if cloning a repo will cause the build machine to get stuck. It happened with `git checkout <git commit>`

**Review by skupr-anaconda — APPROVED** [review]
LGTM if `git` on `win64` is acceptable

**Review by psteyer — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:73]: I guess you mean `# [not win]` ?

Comment (skupr-anaconda) [reply recipe/meta.yaml:73]: Yeah, sorry for confusion, I meant  `# [not win]`

**Pull request merged**